docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/checkbox/checkbox.stories.js

```diff
@@ -0,0 +1,32 @@
+// phpcs:ignoreFile
+import { radios, text } from '@storybook/addon-knobs';
+import CivicThemeCheckbox from './checkbox.twig';
+
+export default {
+  title: 'Atoms/Checkbox',
+  parameters: {
+    layout: 'centered',
+  },
+};
+
+export const Checkbox = (knobTab) => {
+  const generalKnobTab = typeof knobTab === 'string' ? knobTab : 'General';
+
+  const generalKnobs = {
+    theme: radios(
```

Comment:
not all of the props were mapped to the props in twig file

and some of the props were not removed from the twig file